doc/source/fault-tolerance.rst

```diff
@@ -88,3 +88,87 @@ You can experiment with this behavior by running the following code.
             print(counter)
         except ray.exceptions.RayActorError:
             print('FAILURE')
+
+Actor Checkpointing
+~~~~~~~~~~~~~~~~~~~
+
+It's often important to be able to checkpoint the state of an actor in order to
```

Comment:
nit: provide an actionable message - "In event of failure, you can use the ``ray.actor.Checkpointable`` to recover actors to a particular state. Here are some common situations where this is useful: "